apps/system/js/utility_tray.js

```diff
@@ -278,408 +343,13 @@ window.UtilityTray = {
-    var y = touch.pageY;
-    var dy = -(this.startY - y);
-    var move = dy - this.lastDelta;
-    var moved = Math.abs(move) > 0;
-
-    this.validateCachedSizes();
-    this.overlay.classList.add('visible');
-
-    if (moved) {
-      this.lastMoveTime = timestamp;
-      this.lastMove = move;
-    }
-
-    this.lastDelta = dy;
-
-    // It is not longer a tap if
-    // the finger moves over 4px.
-    if (Math.abs(dy) > 4 && this.isTap) {
-      this.debug('not a tap');
-      this.publish('-overlayopening');
-      this.isTap = false;
-    }
-
-    // Don't move the tray until we
-    // know the gesture is a swipe.
-    if (this.isTap) {
-      this.debug('still a tap', this.startY, y);
-      return;
-    }
-
-    if (this.shown) {
-      dy += screenHeight;
-    }
-
-    // When the tray has gone beyond it's maximum,
-    // reset the start position so that when the
-    // finger is dragged up, the tray responds.
-    if (this.shown && dy > screenHeight) {
-      this.startY = y;
-    }
-
-    // Clamp the y position within
-    // the vertical screen bounds
-    dy = Math.max(0, dy);
-    dy = Math.min(screenHeight, dy);
-
-    // Stick the tray to the finger
-    var style = this.overlay.style;
-    style.transition = '';
-    style.transform = 'translateY(' + dy + 'px)';
-
-
-    // As the tray moves with the finger, we keep
-    // the actual content of the tray at the same
-    // position on the screen. This means that you
-    // can see the content of the tray without
-    // the tray having to be fully open.
-    this.notifications.style.transition = '';
-    this.notifications.style.transform =
-      'translateY(' + (this.screenHeight - dy) + 'px)';
-
-    // Hides the real status bar and adds a -moz-element
-    // projection of #status-bar into the utility tray
-    this.screen.classList.add('utility-tray-in-transition');
-    this.transitioning = true;
-  },
-
-  onTouchEnd: function ut_onTouchEnd(evt) {
-    this.debug('touch end');
-    evt.stopImmediatePropagation();
-
-    if (!this.active) {
-      return;
-    }
-
-    var touch = evt.changedTouches[0];
-    this._triggerSearchOnTouchEnd(touch);
-    this.endMove(touch, evt.timeStamp);
-  },
-
-  endMove: function(touch, timestamp) {
-    this.debug('end move');
-
-    // Hide utility tray while
-    // the screen is switched off.
-    if (Service.query('locked')) {
-      this.hide(true);
-    } else {
-
-      // We don't do any animation when the tray
-      // is down and the user swipes down.
-      var down = this.lastDelta > 0;
-      var ignore = this.shown && down || this.isTap;
-
-      if (!ignore) {
-        var timeDelta = timestamp - this.lastMoveTime;
-        var action = this.getAction(touch.pageY);
-        this.animationTime = this.getAnimationTime(timeDelta);
-        this[action]();
-      } else {
-        this.debug('no transition', down, this.shown, this.isTap);
-        this.transitioning = false;
-      }
-    }
-
-    this.active = false;
```

Comment:
looks like there's still some code checking `this.active` or `utilityTray.active` but we're not maintaining it anymore